packages/kit/src/hooks/useReferFriends.tsx

```
@@ -46,20 +46,7 @@ export function useToReferFriendsModalByRootNavigation() {
-    );
-    if (mainRoute?.state?.routes) {
-      return mainRoute.state.routes.find(
-        (route) => route.name === ETabRoutes.ReferFriends,
-      );
-    }
-  }
-  return false;
-};
+
 
 export const useReferFriends = () => {
```

ESLint

[error] Delete `⏎⏎`

(prettier/prettier)